tests/pytests/unit/modules/test_chroot.py

```diff
@@ -0,0 +1,345 @@
+#
```

Comment:
please remove all of these comments. line 1-19

Reply:
removed